src/core/HyperlanePermissionlessDeployer.ts

```diff
@@ -100,54 +109,81 @@ export class HyperlanePermissionlessDeployer {
     );
   }
 
-  get chains(): ChainName[] {
+  get skipLocalDeploy(): boolean {
```

Comment:
On the topic of class accessors in JS: I don't feel super strongly but note that some organizations avoid them entirely (e.g. https://github.com/airbnb/javascript/issues/1880)
They're part of a long list of JS features that some experts consider 'bad', so I typically avoid them. No need to rewrite, but something to think about.

Reply:
interesting, didn't know this but it makes sense. Changed